coral/package.json

```diff
@@ -75,30 +75,30 @@
     "@testing-library/user-event": "^14.4.3",
     "@types/jest": "^29.5.1",
     "@types/lodash": "^4.14.194",
-    "@types/node": "*",
+    "@types/node": "^18.17.14",
     "@types/react": "^18.2.0",
     "@types/react-dom": "^18.2.1",
     "@types/testing-library__jest-dom": "^5.14.6",
     "@typescript-eslint/eslint-plugin": "^5.59.1",
     "@typescript-eslint/parser": "^5.59.1",
-    "@vitejs/plugin-react": "^4.0.0",
+    "@vitejs/plugin-react": "^4.0.4",
     "eslint": "^8.39.0",
     "eslint-config-prettier": "^8.8.0",
     "eslint-import-resolver-typescript": "^3.5.5",
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-lodash": "^7.4.0",
-    "eslint-plugin-no-relative-import-paths": "^1.4.0",
+    "eslint-plugin-no-relative-import-paths": "^1.5.2",
     "eslint-plugin-react": "^7.32.2",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^29.5.0",
-    "jest-environment-jsdom": "^29.5.0",
+    "jest": "^29.6.4",
+    "jest-environment-jsdom": "^29.6.4",
     "jest-transform-stub": "^2.0.0",
     "lint-staged": "^13.0.3",
     "lodash": "^4.17.21",
     "msw": "^0.47.4",
     "openapi-typescript": "^6.2.0",
     "prettier": "^3.0.0",
-    "prop-types": ">=15.6.0 <16.0.0",
+    "prop-types": "^15.8.1",
```

Comment:
Why do we need the `prop-types` dependency? :o

Reply:
It's a missing dev dependency of a package we're using and pnpm encourages you :D to install those explicitly. I sometimes check that and see where we can remove those or have to add new ones.